src/services/point.service.ts

```diff
@@ -54,4 +110,12 @@ export class PointService {
     const allPoints = await PointsModel.find({ guildID: this._guild.id });
     return allPoints.sort((a, b) => b.numPoints - a.numPoints)[allPoints.length - 1] as IUserPoints;
   }
+
+  // returns how long since the user was last crowned king
+  public async getCrownedTime(id: string): Promise<number> {
```

Comment:
What is the precision of this? Is there an edge case where we return something stupid like “crowned for 3us”

Reply:
the edge case would make the footer say "...crowned for 0 ms" but ive added a check to only show times greater than 1 second